src/PeerPoolMgr.cc

```diff
@@ -225,7 +224,25 @@ PeerPoolMgr::checkpoint(const char *reason)
 void
 PeerPoolMgr::Checkpoint(const Pointer &mgr, const char *reason)
 {
-    CallJobHere1(48, 5, mgr, PeerPoolMgr, checkpoint, reason);
+    CallService((mgr.valid() ? mgr->context : nullptr), [&] {
```

Comment:
Since we have to check `mgr` validity, let's avoid calling CallJobHere1() with an invalid `mgr` _unless_ calling CallJobHere1() with an invalid `mgr` still prints `reason` (with debug_options ALL,9) when refusing to schedule or fire the call.

In other words, if skipping CallJobHere1() does not remove `reason` debugging, then let's skip CallJobHere1().

Reply:
I could not find any UnaryMemFunT::print() for nil job arguments.